modules/040-node-manager/hooks/handle_node_templates.go

```diff
@@ -295,6 +302,15 @@ func applyNodeTemplate(nodeObj *v1.Node, node, nodeGroup NodeSettings) error {
 	}
 	newLabels["node.deckhouse.io/type"] = nodeGroup.NodeType.String()
 
+	// 1.4 Add label that indicates that we've reserved memory
+	if nodeGroup.MemoryReserved {
+		_, ok = newLabels["node.deckhouse.io/memory-reserved"]
+		if !ok {
+			labelsChanged = true
+		}
+		newLabels["node.deckhouse.io/memory-reserved"] = ""
+	}
+
```

Comment:
In the discussion we determined that the user should turn off early-oom, so we do not need to do this.

Reply:
Fixed.